plugins/fr.inria.diverse.melange/src/main/java/fr/inria/diverse/melange/processors/LanguageBuilder.xtend

```diff
@@ -325,12 +325,12 @@ class LanguageBuilder extends DispatchMelangeProcessor{
 	/**
 	 * Search in {@link rootPackage} for EClass named as in {@link classes}
 	 */
-	private def List<EModelElement> getClasses(EPackage rootPackage, List<String> classes){
+	private def List<EModelElement> getClasses(EPackage rootPackage, List<EClass> classes){
 		//TODO: manage sub packages
 		val res = new ArrayList<EModelElement>()
 		
 		rootPackage.EClassifiers.filter(EClass).forEach[cl|
-			if(classes.contains(cl.name)) res.add(cl)
+			if(classes.contains(cl)) res.add(cl)
```

Comment:
contains() is based on equals() (reference equality), so this probably won't work. A simple name matching should do the job.